tcg/tcg-op-vec.c

```diff
@@ -734,16 +986,16 @@ void tcg_gen_bitsel_vec(unsigned vece, TCGv_vec r, TCGv_vec a,
     tcg_debug_assert(bt->base_type >= type);
     tcg_debug_assert(ct->base_type >= type);
 
-    if (TCG_TARGET_HAS_bitsel_vec) {
+    /* if (TCG_TARGET_HAS_bitsel_vec) {
```

Comment:
Can you explain why this code is commented, and if it needs to be preserved as a comment in the commit? Is it not working on some target arch?